imagersite/imager_images/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils import timezone
+
+
+PRIVACY = [('PR', 'private'),
+           ('SH', 'shared'),
+           ('PU', 'public')]
+
+
+@python_2_unicode_compatible
+class Photo(models.Model):
+    file = models.ImageField(upload_to='photo_files/%Y-%m-%d')
+    owner = models.ForeignKey(User, null=False)
+    title = models.CharField(max_length=128)
+    description = models.TextField()
+    date_uploaded = models.DateTimeField(default=timezone.now)
```

Comment:
This default value will only come into play if you don't provide another value.  Watch out for that.  It might be better to use the `auto_now` or `auto_now_save` arguments to the field constructor